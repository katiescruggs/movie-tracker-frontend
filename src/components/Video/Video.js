import React from 'react' 
import './Video.css'

const FilmReel = () => {
  return (
      <div className="film-reel-container">
      <iframe src="https://www.youtube.com/embed/9RTaIpVuTqE?rel=0&amp;autoplay=1&amp;controls=0&amp;showinfo=0" frameborder="0" allowfullscreen></iframe>
      </div>
  )
}

export default FilmReel;
