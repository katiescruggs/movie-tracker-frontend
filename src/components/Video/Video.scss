.film-reel-container {
  animation: Show 6.2s;
  visibility: hidden;
}

iframe {
  position: absolute;
  top: 100px;
  left: 0;
  z-index: 10;
  width: 100%;
  height: calc(100% - 100px);
}

@keyframes Show {
  0% {
    visibility: visible;
  }
  100% {
    visibility: hidden;
  }
}
