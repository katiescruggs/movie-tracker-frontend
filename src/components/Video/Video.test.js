import React from 'react';
import { shallow } from 'enzyme';
import Video from './Video';

describe('Video tests', () => {
  let video;

  beforeEach(() => {
    video = shallow(<Video />);
  });

  it('should be defined', () => {
    expect(video).toBeDefined();
  });

  it('should match the snapshot', () => {
    expect(video).toMatchSnapshot();
  });
});