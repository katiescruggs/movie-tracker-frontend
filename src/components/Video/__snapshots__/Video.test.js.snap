// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Video tests should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FilmReel />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <iframe
        allowFullScreen={true}
        frameBorder="0"
        src="https://www.youtube.com/embed/9RTaIpVuTqE?rel=0&autoplay=1&controls=0&showinfo=0"
/>,
      "className": "film-reel-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "allowFullScreen": true,
        "frameBorder": "0",
        "src": "https://www.youtube.com/embed/9RTaIpVuTqE?rel=0&autoplay=1&controls=0&showinfo=0",
      },
      "ref": null,
      "rendered": null,
      "type": "iframe",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <iframe
          allowFullScreen={true}
          frameBorder="0"
          src="https://www.youtube.com/embed/9RTaIpVuTqE?rel=0&autoplay=1&controls=0&showinfo=0"
/>,
        "className": "film-reel-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "allowFullScreen": true,
          "frameBorder": "0",
          "src": "https://www.youtube.com/embed/9RTaIpVuTqE?rel=0&autoplay=1&controls=0&showinfo=0",
        },
        "ref": null,
        "rendered": null,
        "type": "iframe",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
